alarm/binance_utils.py

```diff
@@ -44,4 +43,4 @@ def parse_candle_from_websocket_update(data):
     trade_pair = candle['s']
     trade_pair_lower_case = trade_pair.lower()
 
-    return candle_high_price, candle_low_price, trade_pair_lower_case
+    return candle_high_price, candle_low_price, trade_pair_lower_case
```

Comment:
Pay attention to this. PEP-8 requires an empty line in the end of the file.

Reply:
++